packages/celotool/src/cmds/bots/auto-verify.ts

```diff
@@ -70,6 +73,16 @@ export const builder = (yargs: Argv) => {
       description: 'The index of the account to use',
       default: 0,
     })
+    .option('salt', {
+      type: 'string',
+      description: 'The salt to use instead of getting a pepper from ODIS',
+      default: '',
+    })
+    .option('context', {
```

Comment:
can we not get this from the node we are connected to? or do we want the option for those to not be in sync?

if we don't already have a network ID --> human readable name, feels like we should.

Reply:
Yeah would be nice to get a better description of this arg

Reply:
this will be run in a Helm chart which can easily pass this in as a value (as soon as it's deployed not on baklava)